dev/io.openliberty.mail.2.1.internal_fat/build.gradle

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+/* Define greenmail dependencies */
+dependencies {
+  requiredLibs project(':com.ibm.ws.org.slf4j.api'),
+               project(':com.ibm.ws.org.slf4j.jdk14'),
+               'org.eclipse.angus:angus-mail:1.0.0',
+               'com.ibm.ws.org.apache.yoko:yoko-spec-corba:1.5.+',
```

Comment:
```suggestion
               project('com.ibm.ws.org.apache.yoko.corba.spec.1.5'),
```

Reply:
We were not exporting any packages from that bundle; so, it doesn't seem possible to use bundle in this case. But I'l check for other bundles.